README.md

```diff
@@ -11,6 +11,8 @@
 | --------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `--onSuccess COMMAND`             | Executes `COMMAND` on **every successful** compilation.                                                                                             |
 | `--onFirstSuccess COMMAND`        | Executes `COMMAND` on the **first successful** compilation.                                                                                         |
+| `--onEmit COMMAND`                | Executes debounced `COMMAND` on **every emitted file**.                                                                                             |
```

Comment:
on every emitted file? are you sure?
Does this means that if I have a complication of 100 files, this command will run 100 times?

Reply:
Also, maybe provide a use case for when someone will want to use onEmit?

Reply:
> Does this means that if I have a complication of 100 files, this command will run 100 times?

No because of `--onEmitDebounce`. Just added a note in the README for this.

> Also, maybe provide a use case for when someone will want to use onEmit?

Just added more to this line